### PR TITLE
questions

### PR DESCRIPTION
**Pull request by danid-CB**

### CODE REVIEW

**Review by theemmarios — APPROVED** [review]
LGTM!

File: e2e/tests/questionOne.test.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { test, expect } from '@playwright/test'
+const URL = 'http://localhost:5173/'
+test('Should be able to click on link for Charmander and verify weaknesses', async ({
+  page,
+}) => {
+  await page.goto(URL)
+  await page.getByTestId('on/off-button').click()
+  await page.getByRole('button', { name: 'Click to Start' }).click()
+  // Should be able to click on link for Charmander and verify weaknesses
```
Comment (kyangcb) [inline e2e/tests/questionOne.test.ts:9]: maybe add a test prompt here // WRITE YOUR TEST HERE

File: e2e/tests/questionTwo.test.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { test, expect } from '@playwright/test'
+const URL = 'http://localhost:5173/pokemon/2'
+test('Should be able to verify correct src of image for Ivysaur', async ({
+  page,
+}) => {
+  await page.goto(URL)
+  const Ivysaur = await page.getByRole('img').first()
```
Comment (kyangcb) [inline e2e/tests/questionTwo.test.ts:7]: maybe add a test prompt here // WRITE YOUR TEST HERE

Comment (kyangcb) [reply e2e/tests/questionTwo.test.ts:7]: Should we be specify what exactly needs to be tested?  Like, test filtering and sorting...

File: e2e/tests/questionOne.test.ts (hunk 1 above)
Comment (theemmarios) [inline e2e/tests/questionOne.test.ts:9]:
```suggestion
  // Should be able to click on link for Charmander and verify weaknesses
  // WRITE YOUR TEST HERE
```

File: e2e/tests/questionTwo.test.ts (hunk 2 above)
Comment (theemmarios) [inline e2e/tests/questionTwo.test.ts:7]:
```suggestion
  const Ivysaur = await page.getByRole('img').first()
  // Should be able to verify correct src of image for Ivysaur
  // WRITE YOUR TEST HERE
```

Comment (danid-CB) [reply e2e/tests/questionTwo.test.ts:7]: added in a comment to write your test here

File: e2e/tests/questionOne.test.ts (hunk 1 above)
Comment (danid-CB) [reply e2e/tests/questionOne.test.ts:9]: made changes

File: e2e/tests/questionTwo.test.ts (hunk 2 above)
Comment (danid-CB) [reply e2e/tests/questionTwo.test.ts:7]: made changes

File: e2e/tests/questionOne.test.ts (hunk 1 above)
Comment (danid-CB) [reply e2e/tests/questionOne.test.ts:9]: added in a comment to write your test here

**Pull request merged**